operations/mimir-mixin/dashboards/tenants.libsonnet

```diff
@@ -159,6 +159,67 @@ local filename = 'mimir-tenants.json';
       ),
     )
 
+    .addRow(
+      $.row('Ingestion requests')
+      .addPanel(
+        local title = 'Distributor requests incoming rate';
+        $.panel(title) +
+        $.queryPanel(
+          'sum(rate(cortex_distributor_requests_in_total{%(job)s, user="$user"}[$__rate_interval]))'
+          % { job: $.jobMatcher($._config.job_names.distributor) },
+          'rate',
+        ) +
+        { legend: { show: false } } +
+        $.panelDescription(
+          title,
+          |||
+            The rate of requests that have come in to the distributor, including rejected requests.
+          |||
+        ),
+      )
+      .addPanel(
+        local title = 'Distributor requests received (accepted) rate';
+        $.panel(title) +
+        $.queryPanel(
+          [
+            'sum(rate(cortex_distributor_received_requests_total{%(job)s, user="$user"}[$__rate_interval]))'
+            % { job: $.jobMatcher($._config.job_names.distributor) },
+            user_limits_overrides_query('request_rate'),
+          ],
+          [
+            'rate',
+            'limit',
```

Comment:
In the screenshot that you posted it doesn't show the limit, is the screenshot outdated or did the limits query not work?

Reply:
We don't have limits for this actually in use anywhere (and likely won't except for the free tier due to Reasons)